java-squid/src/test/java/org/sonar/java/SquidUserGuideTest.java

```diff
@@ -144,7 +144,8 @@ public void measures_on_project() throws Exception {
     assertThat(metrics.get("statements").intValue()).isEqualTo(12047);
     assertThat(metrics.get("complexity").intValue()).isEqualTo(8475 - 80 /* SONAR-3793 */- 2 /* SONAR-3794 */);
     assertThat(metrics.get("comment_lines").intValue()).isEqualTo(17908);
-    assertThat(metrics.get("public_api").intValue()).isEqualTo(3221);
+    // 69: SONARJAVA-861 analyseAccessors property of the measurer is set to true. Getters and setters ignored.
+    assertThat(metrics.get("public_api").intValue()).isEqualTo(3221 - 69);
```

Comment:
We should probably have two tests: one with the value of the property set to true and the other one to false, to actually see the difference.